granary/meetup.py

```diff
@@ -86,9 +93,9 @@ def _create(self, obj, preview=False, include_link=source.OMIT_LINK, ignore_form
             return source.creation_result(description=desc)
 
         create_resp = {
-                'url': '%(event_url)s#rsvp-by-%(user_id)s' % {
+                'url': '%(event_url)s#rsvp-by-%(url)s' % {
```

Comment:
thanks for url-escaping! this is still a bit painful and ugly though. remind me, it's because we don't always get the numeric id from their API? if so, how about we drop the leading `https://www.meetup.com/`, since we know that at least will always be there?

Reply:
We get the numeric ID from the user info endpoint, but I'm not sure how we'd hook it in here.

The reason I've had to do this was that Bridgy sends the actor as ie my profile url which I wasn't sure how to map to the numeric ID

Reply:
got it, thanks. so i guess we could fetch the user info endpoint here. seems doable, but heavy. otherwise, let's use just the URL path.

Reply:
Sure, are you happy if we leave it as-is for now? And maybe enhance in the future?

Reply:
you mean, enhance by dropping `https://www.meetup.com/` from the fragment? seems small with eg `urlparse`, but sure, if you want to add a TODO and do that later, sgtm.

Reply:
Sorry, by enhance I meant replace this exiting method with the `numeric_id` via a user_info call. Would prefer to do as a follow up so we can get this portion over the line!

Reply:
sgtm! thanks for your patience 😁